scripts/base/frameworks/config/main.zeek

```diff
@@ -147,6 +147,9 @@ function format_value(value: any) : string
 
 function config_option_changed(ID: string, new_value: any, location: string): any &is_used
 	{
+	# Some option updates reflect Zeek-internal activity and shouldn't be logged.
+	if ( location == "<skip-config-log>" )
```

Comment:
Is there a place where we could document this?

Reply:
Yeah, I'd say in the config framework docs since they cover change handlers in some detail.

I just noticed that the logs chapter in the manual doesn't cover config.log at all yet.